src/WebJobs.Script.WebHost/PreJIT/linux.coldstart.jittrace

```diff
@@ -6635,6 +5971,640 @@ string.Create<ValueTuple`3<__Canon,char,uint32>>(int32,ValueTuple`3<__Canon,char
 [System.Web.HttpUtility]System.Web.Util.Utf16StringValidator.ValidateString(string)~System.Web.Util.Utf16StringValidator,System.Web.HttpUtility~1~0~ValidateString
 char.IsSurrogate(char)~System.Char,System.Private.CoreLib~1~0~IsSurrogate
 int64.GetHashCode()~System.Int64,System.Private.CoreLib~0~0~GetHashCode
-string.Create<int64>(int32,int64,SpanAction`2<char,int64>)~System.String,System.Private.CoreLib~3~1:System.Int64,System.Private.CoreLib~Create
 string.Ctor(char[])~System.String,System.Private.CoreLib~1~0~Ctor
 string.EndsWith(char)~System.String,System.Private.CoreLib~1~0~EndsWith
+[dotprintenv]dotprintenv.getenv.Run(HttpRequest,ILogger)~dotprintenv.getenv,dotprintenv~2~0~Run

```

Comment:
lines starting with [dotprintenv] should be removed. i think currently that is just mentioned as a manual step in the instructions. but should probably go into one of the tools for next time.

Reply:
done!